pkg/env/node_scan.go

```diff
@@ -5,4 +5,16 @@ import "time"
 var (
 	// NodeRescanInterval will set the duration for when to fetch node inventory to be scanned for vulnerabilities
 	NodeRescanInterval = registerDurationSetting("ROX_NODE_RESCAN_INTERVAL", 4*time.Hour)
+
+	// NodeInventoryCacheDuration defines the time after which a cached inventory is considered outdated
+	NodeInventoryCacheDuration = registerDurationSetting("ROX_NODE_INVENTORY_CACHE_TIME", 10*time.Minute)
+
+	// NodeInventoryInitialBackoff defines the initial time in seconds a Node Inventory will be delayed if a backoff file is found
+	NodeInventoryInitialBackoff = RegisterIntegerSetting("ROX_NODE_INVENTORY_INITIAL_BACKOFF", 30)
```

Comment:
Maybe it makes sense to use typed values in other places too. Here - `30*time.Second`, and similarly for the max_backoff?

Reply:
Good idea! I have set this up as durations as well.